cardinal/system/system_game_setup.go

```diff
@@ -198,6 +227,112 @@ func (s *GameSetup) setupMountainPath(world cardinal.WorldContext) {
+
+	roomID := s.RoomStore.Add(component.Room{
+		Description: "a dusty forge",
+		RoomType:    enums.RoomTypeForge,
+	})
+
+	tidForge := s._textGuid("a dusty forge")
+	txtForge := (" and you can see that it has not been used in ages." +
+		" There are many blood spots accross the forge and even the anvil is broken." +
+		" You don't know what happened here.")
+
+	// Logs to verify the ID's in the setup
+	world.Logger().Debug().Msgf("Object IDs for room %d: %v", roomID, [32]uint32{})
+	world.Logger().Debug().Msgf("Directional Object IDs for room %d: %v", roomID, dObjs)
+
+	// Inspect Data Passed to Setup Functions
+	world.Logger().Debug().Msgf("Data passed to createPlace for room %d: %+v", roomID, [32]uint32{})
+	world.Logger().Debug().Msgf("Data passed to createDirectionalObject for room %d: %+v", roomID, dObjs)
+
+	s.createPlace(roomID, enums.RoomTypeForge, dObjs, objs, tidForge, txtForge, world)
+
+	world.Logger().Debug().Msg("---->Forge setup complete")
+}
+
+// setupCellar sets up the cellar room in the game world.
+func (s *GameSetup) setupCellar(world cardinal.WorldContext) {
+	// KCELLAR -> UP
+	world.Logger().Debug().Msg("---->Setting up the cellar.....")
+
+	open2Barn := s.createAction(enums.ActionTypeOpen, " The way is opened"+
+		" just don't fall when going up", true, true, true, 0, 0)
+	plainBarn := []uint32{open2Barn}
+	dObjs := []component.Object{s.createDirObject(enums.DirectionTypeUp, enums.RoomTypeBarn,
+		enums.ObjectTypeTrapdoor, enums.MaterialTypeStone,
+		"trapdoor", plainBarn, world)}
+
+	// Dynamite Object
+	lightDynamite := s.createAction(enums.ActionTypeLight, " Seeing the fuse sparkling makes you remember the fireworks you loved as a child."+
+		" Suddenly, you return back and see that there is almost no time."+
+		" You have to do something or you will be flying into meat pieces!", true, false, true, 0, 0)
+
+	throwDynamite := s.createAction(enums.ActionTypeThrow, " You throw quickly the dynamite"+
+		" and start running for the hills as your life depens on it", true, false, true, 0, 0)
+
+	dynamiteActions := []uint32{lightDynamite, throwDynamite}
+	objs := []component.Object{s.createObject(enums.ObjectTypeDynamite, enums.MaterialTypeIKEA,
+		"a high quality dynamite that can blast almost everything. It needs to be lighted up first", enums.ObjectTypeDynamite.String(), dynamiteActions, true, world)}
+
+	// GLUE Object
+	sniff := s.createAction(enums.ActionTypeSniff, " The smell of it is really relaxing,"+
+		" but for now thats all", true, false, true, 0, 0)
+
+	glueActions := []uint32{sniff}
+	objs = append(objs, s.createObject(enums.ObjectTypeGlue, enums.MaterialTypeShit,
+		"some glue of a strange name: &$#${+>@#!@. Don't knowing if it's your imagination, it is calling you", enums.ObjectTypeGlue.String(), glueActions, true, world))
```

Comment:
"some oddly names glue. Not knowing if its your imagination it seems to be calling you"

Reply:
done